apps/pollbook/frontend/src/api.ts

```diff
@@ -178,7 +178,7 @@ export const searchVoters = {
     return useQuery(
       this.queryKey(searchParams),
       () => apiClient.searchVoters({ searchParams }),
-      { refetchOnMount: 'always' }
+      { refetchInterval: DEFAULT_QUERY_REFETCH_INTERVAL }
```

Comment:
the refetchOnMount "always" is already the default behavior when staleTime is 0 so that is not needed here.